### ISSUE TITLE
Error when using from Electron 

### ISSUE THREAD

**Issue by anjackson** [jarofghosts/memento-client#8]
I just tried to use this from an Electron app, and got this error. Any ideas?

```
TypeError: value.charAt is not a function
    at skipSpaces (/Users/andy/Documents/workspace/ukwa-player/node_modules/http-link/main.js:18:55)
    at httpLink.parse (/Users/andy/Documents/workspace/ukwa-player/node_modules/http-link/main.js:130:15)
    at parseLinks (/Users/andy/Documents/workspace/ukwa-player/node_modules/memento-client/index.js:68:18)
    at ConcatStream.<anonymous> (/Users/andy/Documents/workspace/ukwa-player/node_modules/concat-stream/index.js:36:43)
    at emitNone (events.js:91:20)
    at ConcatStream.emit (events.js:185:7)
    at finishMaybe (/Users/andy/Documents/workspace/ukwa-player/node_modules/concat-stream/node_modules/readable-stream/lib/_stream_writable.js:607:14)
    at endWritable (/Users/andy/Documents/workspace/ukwa-player/node_modules/concat-stream/node_modules/readable-stream/lib/_stream_writable.js:615:3)
    at ConcatStream.Writable.end (/Users/andy/Documents/workspace/ukwa-player/node_modules/concat-stream/node_modules/readable-stream/lib/_stream_writable.js:571:41)
    at IncomingMessage.onend (_stream_readable.js:511:10)
```

**Comment by jarofghosts** [comment]
Published @2.0.4 that should fix the issue, let me know!

**Comment by anjackson** [comment]
Looks great, thanks!
